vars/multicloudDeploy.groovy

```diff
@@ -123,8 +123,7 @@ void call(parameters = [:]) {
                         deployType: deploymentType,
                         cloudFoundry: target,
                         mtaExtensionDescriptor: target.mtaExtensionDescriptor,
-                        mtaExtensionCredentials: target.mtaExtensionCredentials,
-                        mtaPath: script.commonPipelineEnvironment.mtarFilePath,
```

Comment:
I guess `mtaPath` has been removed here intentionally since we use the value provided from cpe.mtarFilePath anyway

Reply:
yes